catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/utils/SeekbarsPickerWrapper.java

```diff
@@ -0,0 +1,106 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.uiespresso.content.brick.utils;
+
+import android.support.test.espresso.action.ViewActions;
+import android.view.View;
+
+import org.catrobat.catroid.R;
+import org.catrobat.catroid.uiespresso.util.wrappers.ViewInteractionWrapper;
+import org.hamcrest.Matcher;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+
+public final class SeekbarsPickerWrapper extends ViewInteractionWrapper {
+	public static final float MAX_COLOR_SEEKBAR_VALUE = 255.0f;
+	public static final float MAX_PHIRO_SEEKBAR_VALUE = 100.0f;
+	public static final float MIN_SEEKBAR_VALUE = 0.0f;
+	public static final Matcher<View> COLOR_SEEKBAR_MATCHER_RED = withId(R.id.color_rgb_seekbar_red);
+	public static final Matcher<View> COLOR_SEEKBAR_MATCHER_GREEN = withId(R.id.color_rgb_seekbar_green);
+	public static final Matcher<View> COLOR_SEEKBAR_MATCHER_BLUE = withId(R.id.color_rgb_seekbar_blue);
+	public static final Matcher<View> PHIRO_SEEKBAR_MATCHER = withId(R.id.single_seekbar_seekbar);
+
+	private SeekbarsPickerWrapper(Matcher<View> viewMatcher) {
+		super(onView(viewMatcher));
+	}
+
+	public static SeekbarsPickerWrapper onColorSeekbar() {
+		return new SeekbarsPickerWrapper(COLOR_SEEKBAR_MATCHER_RED);
+	}
+
+	public static SeekbarsPickerWrapper onSingleSeekbar() {
+		return new SeekbarsPickerWrapper(PHIRO_SEEKBAR_MATCHER);
+	}
+
+	public SeekbarsPickerWrapper performSwipeRedSeekbar(float valueToBeEntered) {
+		onView(COLOR_SEEKBAR_MATCHER_RED).perform(CustomSwipeAction.swipeToPosition(valueToBeEntered));
+		return this;
+	}
+
+	public SeekbarsPickerWrapper performSwipeBlueSeekbar(float valueToBeEntered) {
+		onView(COLOR_SEEKBAR_MATCHER_BLUE).perform(CustomSwipeAction.swipeToPosition(valueToBeEntered));
+		return this;
+	}
+
+	public SeekbarsPickerWrapper performSwipeGreenSeekbar(float valueToBeEntered) {
+		onView(COLOR_SEEKBAR_MATCHER_GREEN).perform(CustomSwipeAction.swipeToPosition(valueToBeEntered));
+		return this;
+	}
+
+	public SeekbarsPickerWrapper performSwipePhiroSeekbar(float valueToBeEntered) {
+		onView(PHIRO_SEEKBAR_MATCHER).perform(CustomSwipeAction.swipeToPosition(valueToBeEntered));
+		return this;
+	}
+
+	public SeekbarsPickerWrapper performUndo() {
+		onView(ActionMenu.UNDO)
+				.perform(click());
+		return this;
+	}
+
+	public SeekbarsPickerWrapper performRedo() {
+		onView(ActionMenu.REDO)
+				.perform(click());
+		return this;
+	}
+
+	public void pressBack(Matcher<View> viewMatcher) {
```

Comment:
this is unnecessary, theres also a espresso action "pressBack()" that does not need a view to operate on.